core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -9489,85 +9489,100 @@ public Sql schema(CalciteAssert.SchemaSpec schemaSpec) {
   @Test public void testdatetrunc() {
     final RelBuilder builder = relBuilder();
     final RexNode trunc = builder.call(SqlLibraryOperators.TRUNC,
-        builder.literal("2008-19-12"), builder.literal("DAY"));
+        builder.literal("2008-09-12"), builder.literal("DAY"));
     final RelNode root = builder
         .scan("EMP")
         .project(builder.alias(trunc, "FD"))
         .build();
-    final String expectedSql = "SELECT TRUNC('2008-19-12', 'DAY') AS \"FD\"\n"
+    final String expectedSql = "SELECT TRUNC('2008-09-12', 'DAY') AS \"FD\"\n"
         + "FROM \"scott\".\"EMP\"";
-    final String expectedBiqQuery = "SELECT DATE_TRUNC('2008-19-12', DAY) AS FD\n"
+    final String expectedBiqQuery = "SELECT DATE_TRUNC('2008-09-12', DAY) AS FD\n"
+        + "FROM scott.EMP";
+    final String expectedSparkQuery = "SELECT CAST(DATE_TRUNC('DAY', '2008-09-12') AS DATE) FD\n"
```

Comment:
Can you please rename the expectedSparkQuery variable name to expectedSparkSql?

Reply:
We have many places where we are using expectedSparkQuery also.
I guess will be fine we can keep this same name also.

Reply:
ok